adapters/audienceNetwork/audienceNetworktest/exemplary/native-1.1.json

```diff
@@ -0,0 +1,135 @@
+                "native": {
+                    "request": "{\"ver\":\"1.1\",\"context\":1,\"contextsubtype\":11,\"plcmttype\":4,\"plcmtcnt\":1,\"assets\":[{\"id\":1,\"required\":1,\"title\":{\"len\":500}},{\"id\":2,\"required\":1,\"img\":{\"type\":3,\"wmin\":1,\"hmin\":1}},{\"id\":3,\"required\":0,\"data\":{\"type\":1,\"len\":200}},{\"id\":4,\"required\":0,\"data\":{\"type\":2,\"len\":15000}},{\"id\":5,\"required\":0,\"data\":{\"type\":6,\"len\":40}},{\"id\":6,\"required\":0,\"data\":{\"type\":500}}]}",
+                    "ver": "1.1"
+                },
+                "ext": {
+                    "bidder": {
+                        "publisherid": "123",
+                        "placementid": "456"
+                    }
+                }
+            }
+        ],
+        "site": {
+            "domain": "prebid.org",
+            "page": "prebid.org"
+        },
+        "device": {
+            "ip": "152.193.6.74"
+        },
+        "user": {
+            "id": "db089de9-a62e-4861-a881-0ff15e052516",
+            "buyeruid": "v4_bidder_token"
+        },
+        "tmax": 500
+    },
+    "httpcalls": [
+        {
+            "expectedRequest": {
+                "uri": "https://an.facebook.com/placementbid.ortb",
+                "headers": {
+                    "Accept": [
+                        "application/json"
+                    ],
+                    "Auth-Token": [
+                        "test-app-id|test-app-secret"
+                    ],
+                    "Content-Type": [
+                        "application/json;charset=utf-8"
+                    ],
+                    "X-Fb-Pool-Routing-Token": [
+                        "v4_bidder_token"
+                    ]
+                },
+                "body": {
+                    "id": "test-req-id",
+                    "imp": [
+                        {
+                            "id": "test-imp-id",
+                            "native": {
+                                "w": -1,
+                                "h": -1
+                            },
+                            "tagid": "123_456"
+                        }
+                    ],
+                    "ext": {
+                        "appnexus": {
+                            "hb_source": 5
+                        },
+                        "prebid": {}
+                    },
+                    "site": {
+                        "domain": "prebid.org",
+                        "page": "prebid.org",
+                        "publisher": {
+                            "id": "123"
+                        }
+                    },
+                    "device": {
+                        "ip": "152.193.6.74"
+                    },
+                    "user": {
+                        "id": "db089de9-a62e-4861-a881-0ff15e052516",
+                        "buyeruid": "v4_bidder_token"
+                    },
+                    "tmax": 500,
```

Comment:
Can we start with 800?

Reply:
this is just a test response, its based on the test request

Reply:
That's part of the internal tests. The exact time has no affect on production code.